builtins/amp-ad.md

```diff
@@ -103,6 +103,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [I-Mobile](../ads/imobile.md)
 - [Improve Digital](../ads/improvedigital.md)
 - [Industrybrains](../ads/industrybrains.md)
+- [InMobi](../ads/inmobi.md)
```

Comment:
this file is deprecated as will soon be removed.
would you add to `extensions/amp-ad/amp-ad.md`